Azure/sonic-utilities
# Update config/show to include PFC Watchdog commands

**- What I did**
- added wrappers to the pfcwd in config and show utilities
-  updated Command-Reference doc

**- How I did it**

**- How to verify it**
Run commands with the option --verbose and make sure executed commands are ok

**- Previous command output (if the output of a command-line utility has changed)**
Example: 
`pfcwd start Ethernet4 200`
**- New command output (if the output of a command-line utility has changed)**
`config pfcwd start Ethernet4 200`

Full list of mapped commands:
```
config pfcwd start <arguments>
config pfcwd stop
config pfcwd interval <interval_in_ms>
config pfcwd counter_poll <enable/disable>
config pfcwd big_red_switch <enable/disable>
config pfcwd start_default
show pfcwd config
show pfcwd stats
```
